src/network/netdev/netdev-gperf.gperf

```diff
@@ -242,8 +242,8 @@ BatmanAdvanced.BridgeLoopAvoidance,       config_parse_bool,
 BatmanAdvanced.DistributedArpTable,       config_parse_bool,                         0,                             offsetof(BatmanAdvanced, distributed_arp_table)
 BatmanAdvanced.Fragmentation,             config_parse_bool,                         0,                             offsetof(BatmanAdvanced, fragmentation)
 BatmanAdvanced.GatewayMode,               config_parse_batadv_gateway_mode,          0,                             offsetof(BatmanAdvanced, gateway_mode)
-BatmanAdvanced.GatewayBandwithDown,       config_parse_badadv_bandwidth,             0,                             offsetof(BatmanAdvanced, gateway_bandwidth_down)
-BatmanAdvanced.GatewayBandwithUp,         config_parse_badadv_bandwidth,             0,                             offsetof(BatmanAdvanced, gateway_bandwidth_up)
```

Comment:
Ugh... Please keep these settings for backward compatibility...

Reply:
did we ship v248 with it? thought it was new

Reply:
The settings are already shipped in v248... Unfortunately.

Reply:
Hm, but after re-configuring, the wrong options are not documented. Let's merge this as is.